worker/src/logger.js

```diff
@@ -0,0 +1,29 @@
+const { format, transports, createLogger } = require('winston');
+
+const formatFunc = ({
+  level, message, label, timestamp, meta = {},
+}) => `${timestamp} ${label || '-'} ${level}: ${message} ${JSON.stringify(meta)}`;
+
+const stringFormatter = format.combine(
+  format.timestamp(),
+  format.colorize(),
+  format.align(),
+  format.printf(formatFunc),
+);
+
+const jsonFormatter = format.combine(
+  format.timestamp(),
+  format.json(),
+);
+
+const formatter = process.env.LOG_JSON_FORMAT ? jsonFormatter : stringFormatter;
+
+const logger = createLogger({
+  level: process.env.LOG_LEVEL,
+  format: formatter,
+  transports: [
+    new transports.Console(),
+  ],
+});
+
+module.exports = logger;
```

Comment:
We need the audit logger from `src` to track any application errors or security sensitive things (a file being dirty would definitely qualify). This base logger is good for any other status or debug/info logging messages.